pkg/rtc/supervisor/participant_supervisor.go

```diff
@@ -62,10 +62,18 @@ func NewParticipantSupervisor(params ParticipantSupervisorParams) *ParticipantSu
 }
 
 func (p *ParticipantSupervisor) Stop() {
+	if p == nil {
```

Comment:
seems a bit fragile having to add this check everywhere. is it possible for the caller to avoid calling this function when it's not created?

Reply:
Yeah can do. Was making the code at the call site a bit heavier, but can switch to that.

Reply:
Changed in this [commit](https://github.com/livekit/livekit/pull/2277/commits/b5d2094a9afa590c60594336f1c96a95a47c4c0c).